internal/provider/models.go

```diff
@@ -86,6 +86,7 @@ type Dashboard struct {
 	Group         string               `json:"group,omitempty"`
 	Name          string               `json:"name"`
 	SchemaVersion string               `json:"schemaVersion"`
```

Comment:
We don't want to advertise SchemaVersion (it's an internal thing used for dashboard migrations).  Perhaps we need to strip SchemaVersion and Group from Dashobard reads (GETs) too.